<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>981</width>
      <height>721</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_gif">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>981</width>
       <height>621</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="train_bg.qrc">:/train/train.gif</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="topbgcolor_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>630</y>
       <width>1661</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="start_bg">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>160</y>
       <width>541</width>
       <height>401</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="train_bg.qrc">:/train/start_page.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="to_way_lbl">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>260</y>
       <width>391</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>LG Smart UI Bold</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Find Shortest Path</string>
     </property>
    </widget>
    <widget class="QLabel" name="to_meet_lbl">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>380</y>
       <width>391</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>LG Smart UI Bold</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Find Middle Point</string>
     </property>
    </widget>
    <zorder>topbgcolor_2</zorder>
    <zorder>label_gif</zorder>
    <zorder>start_bg</zorder>
    <zorder>to_way_lbl</zorder>
    <zorder>to_meet_lbl</zorder>
   </widget>
   <widget class="QLabel" name="topbgcolor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1651</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>topbgcolor</zorder>
   <zorder>frame</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="train_bg.qrc"/>
 </resources>
 <connections/>
</ui>
